README.md

```diff
@@ -73,6 +73,12 @@ NEXPOSEASSETATTRIBUTOR_CLOUDASSETINVENTORY_HTTPCLIENT_TYPE="default"
 NEXPOSEASSETATTRIBUTOR_CLOUDASSETINVENTORY_HTTPCLIENT_SMART_OPENAPI=""
 ```
 
+This list can be generated by building the binary and passing the `-h` or -`help` flag:
+
+```bash
+go build -o app . && ./app -h
```

Comment:
suggestion: FWIW, you can also `go run main.go -h` and get the same thing.

Reply:
Good call, fixed.